packages/sdk/typescript/human-protocol-sdk/src/enums.ts

```diff
@@ -24,3 +24,8 @@ export enum OrderDirection {
   ASC = 'asc',
   DESC = 'desc',
 }
+
+export enum LeaderCategory {
```

Comment:
Lets use a constat for it like in https://github.com/humanprotocol/human-protocol/blob/b95124839b6f0153784afef5a3399a0c8dc6c157/packages/sdk/typescript/human-protocol-sdk/src/constants.ts#L303-L304 

Also use `machine_learning` and `market_making` as values